workspace/test_configurations.go

```diff
@@ -0,0 +1,74 @@
+package workspace
+
+import "runtime"
+
+type TestConfiguration struct {
+	// The static portion of the test Command, which will be run for every test on this track. Examples include `cargo test` or `go test`.
+	// Might be empty if there are platform-specific versions
+	Command string
+
+	// Windows-specific test command. Mostly relevant for tests wrapped by shell invocations. Falls back to `Command` if we're not running windows or this is empty.
+	WindowsCommand string
+
+	// Some tracks test by running a specific file, such as `ruby lasagna_test.rb`. Set this to `true` to look up and include the name of the default test file(s).
+	AppendTestFiles bool
```

Comment:
I wonder if instead of this we could have a placeholder in the command itself, Something like:

```
jasmine-node --coffee %{test_files}
```

That way you're a little bit more flexible, and you could also support:

```
jasmine-node %{test_files} --coffee 
```

Reply:
I would be fine with a placeholder, but I don't think any of the tracks included test file(s) anywhere besides the last position. I mostly went for simplicity here, but am happy to explore the template as well.

Reply:
Yeah I also don't know of any that don't do that, but I sort of like the extra flexibility. I also feel that the template version is more explicit, where one better sees the command being used.

Reply:
Yeah fair! That should be a straightforward change.